ledger_device_sdk/Cargo.toml

```diff
@@ -14,10 +14,12 @@ ledger_device_sdk = { path = ".", features = ["speculos"] }
 testmacro = { git = "https://github.com/yhql/testmacro"}
 
 [dependencies]
-ledger_secure_sdk_sys = "1.0.1" 
+ledger_secure_sdk_sys = {path = "../ledger_secure_sdk_sys", version = "1.0.2"}
+include_gif = {path = "../include_gif", version = "1.0.1"}
```

Comment:
I'm talking about this, shouldn't we use the version from crates.io?

Reply:
no, I will publish a new version of `include_gif` on `crates.io` once the PR will be merged, it will be version 1.0.1.

Reply:
Even if it is true that the modification in this crate is only relevant when building from a workspace environment. But it should not introduce any regression...

Reply:
same for `ledger_secure_sdk_sys`,  new version will be published on crates.io as soon as the PR is merged

Reply:
And last but not least, version is always checked, even locally, so even when working with local path, it is the same version as the one pulled from crates.io

Reply:
Why not publish before merging with proper versions ?

Reply:
@xchapron-ledger indeed, didn't know about that one. Cool stuff. Let's go.